src/routes/GroupSettings/ModeratorsSettingsTab/ModeratorsSettingsTab.js

```diff
@@ -1,11 +1,13 @@
 import PropTypes from 'prop-types'
```

Comment:
lets change the name of this Tab and File to something else. MemberRoles?  or just Roles? Or RolesAndResponsibilities if we can fit that in here too